### PR TITLE
Improve CLI

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: bin/cli.js [hunk 1]
```diff
@@ -8,26 +8,20 @@ const precinct = require('../index.js');
 const { version } = require('../package.json');
 
 program
+  .arguments('<filename>')
```
Comment (XhmikosR) [inline bin/cli.js:11]: This will make sure a filename is always passed.

**Review by mrjoelkemp — APPROVED** [review]
Thanks!

File: bin/cli.js [hunk 2]
```diff
@@ -8,26 +8,20 @@ const precinct = require('../index.js');
...
-const content = fs.readFileSync(program.args[0], 'utf8');
+  .option('--es6-mixedImports', 'Fetch all dependendies from a file that contains both CJS and ES6 imports')
+  .option('-t, --type <type>', 'The type of content being passed in. Useful if you want to use a non-JS detective')
+  .parse();
 
+const cliOptions = program.opts();
 const options = {
-  es6: {}
+  es6: {
+    mixedImports: cliOptions.es6MixedImports
```
Comment (XhmikosR) [inline bin/cli.js:20]: We could cast this to Boolean BTW

Comment (mrjoelkemp) [reply bin/cli.js:20]: Agreed. Probably safer to do that.

**Pull request merged**